criu/autofs.c

```diff
@@ -331,12 +331,14 @@ static int autofs_revisit_options(struct mount_info *pm)
 {
 	FILE *f;
 	char *str;
+	char *org;
```

Comment:
fgets should get a pointer to the whole buffer:

char *buf;
....
while (fgets(buf, 1024, f)) {
  char *str = buf;